testsuite/MDAnalysisTests/coordinates/test_gro.py

```diff
@@ -187,107 +188,136 @@ def __init__(self):
 
 
 class TestGROReader(BaseReaderTest):
-    def __init__(self, reference=None):
-        if reference is None:
-            reference = GROReference()
-        super(TestGROReader, self).__init__(reference)
+    @staticmethod
+    @pytest.fixture()
+    def ref():
+        return GROReference()
+
+    @staticmethod
+    @pytest.fixture()
+    def reader(ref):
+        reader = ref.reader(ref.trajectory)
+        reader.add_auxiliary('lowf', ref.aux_lowf, dt=ref.aux_lowf_dt, initial_time=0, time_selector=None)
+        reader.add_auxiliary('highf', ref.aux_highf, dt=ref.aux_highf_dt, initial_time=0, time_selector=None)
```

Comment:
Can we find a solution that doesn't require us to repeat this for every child class of BaseReaderTests?